vendors/halo2/src/bn254_shplonk_prover.cc

```diff
@@ -27,20 +31,34 @@ rust::Box<G1JacobianPoint> DoCommit(
       reinterpret_cast<G1JacobianPoint*>(result));
 }
 
-std::vector<math::bn254::Fr> ReadFrs(uint8_t* vec_ptr, size_t i) {
+std::vector<math::bn254::Fr> ReadFrsToRemove(uint8_t* vec_ptr, size_t i) {
   size_t num_bytes = base::EstimateSize(rs::RustVec());
   base::Buffer buffer(&vec_ptr[num_bytes * i], num_bytes);
   rs::RustVec vec;
   CHECK(buffer.Read(&vec));
   return vec.ToVec<math::bn254::Fr>();
 }
 
+PCS::Evals ReadEvalsToRemove(uint8_t* vec_ptr, size_t i) {
+  return PCS::Evals(ReadFrsToRemove(vec_ptr, i));
+}
+
```

Comment:
Do we need this temporary change?

Reply:
This is because there's symbol conflict `ReadEvals`!